addons/point_of_sale/static/tests/tours/utils/payment_screen_util.js

```diff
@@ -279,22 +343,3 @@ export function clickShipLaterButton() {
         },
     ];
 }
-
-export function clickPartnerButton() {
-    return [
-        {
-            content: "click customer button",
-            trigger: "button.partner-button",
-        },
-        {
-            content: "partner screen is shown",
-            trigger: PartnerList.clickPartner().trigger,
-            in_modal: true,
-            run: () => {},
-        },
-    ];
-}
```

Comment:
Why are you removing this? Isn't there a different button to access the list of customers in the payment screen?

Reply:
Does this mean that we don't have a test for changing the customer from the payment screen?

Reply:
I change it because we have ProductScreen.clickPartnerButton that work on payment_screen_tour, it's pretty the same :

`export function clickPartnerButton() {
    return [
        clickReview(),
        {
            content: "click customer button",
            trigger: ".product-screen .set-partner",
        },
        {
            content: "partner screen is shown",
            trigger: PartnerList.clickPartner().trigger,
            in_modal: true,
            run: () => {},
        },
    ];
}`

And, yes, indeed, the 3 call of clickPartnerButton in payment_screen_tour take place on the product screen